azure/Kqlmagic/my_aad_helper.py

```diff
@@ -218,5 +224,21 @@ def acquire_token(self, **options):
             raise AuthenticationError("Unknown authentication method.")
         return self._get_header(token)
 
+
+    def send_email(self, message, mailto):
+
+        port = 465  # For SSL
+        smtp_server = "smtp.gmail.com"
+        sender_email = "dev.kql.test@gmail.com"  # Enter your address
+
+        receiver_email = mailto # Enter receiver address
+
+        password = "Kc0qpELOz8V3"
```

Comment:
is it a permanent password? If it isn't and it must be modified every X days, how we maintain it? anyhow it should be the password of Kqlmagic. To have the password in the code is problemtic, (but I have no clear idea how to avoid it.